### ISSUE TITLE
Device Configuration Error Messages Should be More Useful

### ISSUE THREAD

**Issue by kb1lqc** [FaradayRF/Faraday-Software#82]
We should help make the error responses more specific, granular, and overall more helpful in pinpointing exactly what the user typed in wrong.

If we plan on integrating this with a GUI we should probably think about actual JSON responses that can automate highlighting of erroneous fields.

**Comment by kb1lqd** [comment]
Agreed but let's kick that for a specific campaign sprint. I'd rather move
on and circle back.
